episode-6.srt

```diff
@@ -46,7 +46,7 @@
 10
 00:00:44,100 --> 00:00:45,680
 [ru] В предыдущих сериях:
-[en] Previously on…
+[en] Previously...
```

Comment:
"In the last episode...", "In previous episodes...", or "Previously on Pepper & Carrot..." all work as well. It just doesn't make sense when you read what you had with the next subtitle "Previously on Pepper spent a lot of effort..." because it makes you ask: Previously on what?